components/external-link/global.css

```diff
@@ -1,19 +1,17 @@
-.external::after {
+.external {
   display: inline-flex;
+  column-gap: 0.25em;
+  align-items: center;
 
-  width: 0.75em;
-  height: 0.75em;
+  &::after {
+    width: 1em;
+    height: 1em;
 
-  margin-left: 0.25em;
+    content: "";
 
-  content: "";
+    background-color: currentcolor;
```

Comment:
As already mentioned, using the same color for the external indicator makes it less obvious to identify external links visually.